PSSwagger/PSSwagger.psm1

```diff
@@ -238,7 +240,15 @@ function New-PSSwaggerModule {
         [Parameter(Mandatory = $false, ParameterSetName = 'SpecificationPath')]
         [Parameter(Mandatory = $false, ParameterSetName = 'SpecificationUri')]
         [switch]
-        $ConfirmBootstrap
+        $ConfirmBootstrap,
+
+        [Parameter(Mandatory = $false, ParameterSetName = 'SpecificationPath')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'SpecificationUri')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'SdkAssemblyWithSpecificationPath')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'SdkAssemblyWithSpecificationUri')]
+        [string]
+        [ValidateSet('None', 'PSScriptAnalyzer')]
+        $Formatter = 'None'
```

Comment:
>'None' [](start = 21, length = 6)

Could you please let us know the purpose of 'None' here? If the user doesn't want to format the generated file contents, he will not specify this parameter.

I think, ValidateSet should be [ValidateSet('PSScriptAnalyzer')] and $Formatter param is not assigned with any default value.

Reply:
I think there are 3 options here in total:
1. None
2. PSSA
3. Not specified

When (1) or (2) is specified by the CLI, it should override the config. If 'None' is removed, we can't tell the difference between (1) and (3).